sufia.gemspec

```diff
@@ -60,4 +60,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "factory_girl_rails", '~> 4.4'
   spec.add_development_dependency "equivalent-xml", '~> 0.5'
   spec.add_development_dependency "jasmine", '~> 2.3'
+  spec.add_development_dependency 'rubocop', '~> 0.39'
```

Comment:
I went back and forth on this: should we pin this more tightly to `0.39.0`? The downside is that we'll need to remember to update this on a regular basis, when Rubocop updates. The benefit is that we won't be hit so often with new Rubocop violations (which seems to always happen *after* you've worked up a PR unrelated to Rubocop).